libakumuli/storage_engine/blockstore.h

```diff
@@ -117,37 +117,72 @@ class FixedSizeFileStorage : public BlockStore,
     std::vector<std::string> volume_names_;
 
     //! Secret c-tor.
-    FixedSizeFileStorage(std::string metapath, std::vector<std::string> volpaths);
+    FileStorage(std::string metapath, std::vector<std::string> volpaths);
 
+    virtual void update_current_volume() = 0;
```

Comment:
This is a bit confusing to me because we have update_current_volume and advance_volume. Why can't we just have to different implementations of the `advance_volume`? Or we can come up with a better name for one of the functions.